### PR TITLE
feat(nav): Add z-index on the 'Navigate up' section

### PR DESCRIPTION
**Pull request by gillchristian**
This PR adds a link to the `z-index` in the "Navigate up" section as a quick way to go back to the graph view.

Screenshot 2020-03-23 at 19 04 38

NOTE: I'm not familiar with Rib and the Nueron codebase. Please let me know what should be changed.

### CODE REVIEW

**Comment by humaidq** [comment]
It is nice to have a link to the z-index somewhere, but having it in
the "Navigate up" section doesn't feel right to me. Maybe it would be
make more sense for it to be in the footer?

**Comment by gillchristian** [comment]
@humaidq I didn't find a place to put it that wouldn't require bigger layout/markup changes. IMHO the graph is the top level so I find it intuitive to be part of 'Navigate up'.

That said, I'm fine with it being anywhere in the footer.

There's an open issue for 'Edit on GitHub' (#5), maybe the footer will be extended with more things and a better place from UX point of view.

**Comment by srid** [comment]
I like the idea of including "z-index" in a place like this. I agree that it feels odd to place it in "Navigate up" section (though that would *eventually* make sense once "z-index" becomes part of the graph itself, as a result of #18).

For now, I'd introduce a separate footer section below both Connections/NavigateUp (still inside the same teal box perhaps) and put the z-index in there.

File: src/Neuron/Zettelkasten/View.hs [hunk 1]
```diff
@@ -105,7 +105,11 @@ renderZettel (store, graph) zid = do
         div_ [class_ "column"] $ do
           div_ [class_ "ui header"] "Navigate up"
           let forestB = obviateRootUnlessForest zid $ dfsForestBackwards zid graph
-          ul_ $ renderForest Nothing (LinkTheme_Simple $ Just zid) store graph forestB
+          ul_ $ do
+            li_ $ do
+              a_ [class_ "zettel-link item", href_ "/z-index.html", title_ "z-index"] $ do
+                span_ [class_ "zettel-link-title"] "z-index"
```
Comment (srid) [inline src/Neuron/Zettelkasten/View.hs]: Respecting the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), I'd extract [this code](https://github.com/srid/neuron/blob/44365c6/src/Neuron/Zettelkasten/Link/View.hs#L61-L66) into a function, and then re-use it here.

Comment (gillchristian) [reply src/Neuron/Zettelkasten/View.hs]: Extracted the two links, active and regular. Although active is not a link is used in place of one.

They aren't exactly the same to justify using the same function I think.

**Comment by gillchristian** [comment]
Went for "Others" as the section's title, but I'm not quite happy about it. Wdyt?

Screenshot 2020-03-23 at 23 01 42

**Comment by srid** [comment]
@gillchristian I've made some tweaks to your branch at https://github.com/gillchristian/neuron/pull/1 ... if you could merge that PR, I'll merge this one right away.

**Comment by srid** [comment]
Actually, I've merged it from the command line. Thanks for the PR!

**Pull request merged**